packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -651,6 +651,12 @@ function hookFunctionsSaveWorker(): IWorkflowExecuteHooks {
 					executionId,
 					success: runData.status === 'success',
 					isManual: runData.mode === 'manual',
+					lastNodeExecuted: runData?.data.resultData.lastNodeExecuted,
+					errorNodeType:
+						runData?.data.resultData.error && 'node' in runData?.data.resultData.error
+							? runData?.data.resultData.error.node?.type
+							: undefined,
+					errorMessage: runData?.data.resultData.error?.message.toString(),
```

Comment:
Since we are adding `runData` to this emit in https://github.com/n8n-io/n8n/pull/10232, we might as well add it now already and derive these fields in the `AuditEventRelay`. Would make this clearer and the logic for deriving the fields for the audit events would be in that class